src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -1,240 +1,292 @@
 {% include "src/site/includes/header.html" with drupalTags = true %}
 {% include "src/site/includes/alerts.drupal.liquid" %}
-{% include "src/site/includes/preview-edit.drupal.liquid" %}
-{% include "src/site/includes/breadcrumbs.drupal.liquid" with constructLcBreadcrumbs = true titleInclude = false %}
+
+<!-- Draft status -->
+{% if !entityPublished %}
+<div class="vads-u-background-color--primary-alt-lightest vads-u-padding--1">
+  <div class="vads-l-grid-container medium-screen:vads-u-padding-x--0">
+    You are viewing a draft of "{{ entityUrl.path }}". <a data-same-tab href="{{ drupalSite }}/node/{{ entityId }}/edit">Edit this page in the CMS.</a>
+  </div>
+</div>
+{% endif %}
+
+{% comment %}
+  {% include "src/site/includes/breadcrumbs.drupal.liquid" with constructLcBreadcrumbs = true titleInclude = false %}
+{% endcomment %}
 
 <div id="content" class="interior" data-template="node-campaign_landing_page">
   <main class="va-l-detail-page">
-    <!-- Draft status -->
-    {% if !entityPublished %}
-      <div class="usa-alert usa-alert-info">
-        <div class="usa-alert-body">
-          <p class="usa-alert-text">You are viewing a draft.</p>
-        </div>
-      </div>
-    {% endif %}
 
-    <div class="va-c-hero" style="background-image: url('{{ fieldHeroImage.entity.image.url }}">
+    <!-- HERO-->
+    <div class="va-u-background--image" style="background-image: url('{{ fieldHeroImage.entity.image.url }}">
       <!-- Hero Content -->
-      <div class="usa-grid usa-grid-full">
-        <div class="homepage-hub va-c-hero-content vads-u-padding-x--4 vads-u-padding-y--6">
-          <h1 class="vads-u-color--white">{{ title }}</h1>
-          <hr class="va-c-blue-line vads-u-border-color--primary-alt vads-u-margin-y--2" />
-          <p class="vads-u-color--white">{{ fieldHeroBlurb }}</p>
-
-          {% if fieldPrimaryCallToAction %}
-            <a class="va-c-custom-hero-button usa-button usa-button-secondary vads-u-margin-top--2 vads-u-background-color--white vads-u-border-color--white vads-u-font-size--sm" href="{{ fieldPrimaryCallToAction.entity.fieldButtonLink.url.path }}" rel="noreferrer noopener">
-              {{ fieldPrimaryCallToAction.entity.fieldButtonLabel }}
-            </a>
-          {% endif %}
+      <div class="vads-l-grid-container vads-u-padding-x--0">
+        <div class="vads-l-row">
+          <div class="vads-l-col--12 medium-screen:vads-l-col--6">
+            <div class="va-u-background--gradiant-blue vads-u-padding-x--4 vads-u-padding-y--6  medium-screen:vads-u-margin-right--4">
```

Comment:
Any reason we need to have 4 nested divs here? 🙂 I've seen this elsewhere in the file so I'll just leave 1 comment for this!

Reply:
Ignore the above comment, after pulling it down I think the layout looks really polished now so let's just leave this as is! 🔥

Reply:
Yeah it is kind of a div soup but it's also kind of the nature of a grid framework. The first three levels are just setting up the grid: container -> row -> column -> component. In some cases the column and component styles can be combined but sometimes they might mess with each other